### DISCUSSION TITLE
Cannot install Kubernetes 1.26 with newest Kubeone 1.5.6.  1.26.1 is greater than 1.24

### DISCUSSION THREAD

**Discussion by JWebDev** [kubermatic/kubeone#2637] [Q&A]
``Hi.
Tell me please why I can't install kubernetes 1.26.1 or a little earlier with Kubeone  1.5.6?
https://github.com/kubermatic/kubeone/releases/tag/v1.5.6
The documentation says that with version 1.5.6 you can install 1.26.1 and 1.25, but I have such an error during installation.
```
Error: configuration validation
versions.kubernetes: Invalid value: kubeone.VersionConfig{Kubernetes:"1.26.1"}: kubernetes version does not satisfy version constraint '<= 1.24': 1.26.1 is greater than 1.24. This version is not yet supported. Please refer to the Compatibility section of docs for more details.
```
Thanks!

**Comment by embik** [comment]
Hi @JWebDev, it's understandable that the release note in v1.5.6 is confusing. It refers to a fix that applies to several upstream Kubernetes versions, but not all of them are supported in the 1.5.x release series. For versions supported by KubeOne, please [check our documentation](https://docs.kubermatic.com/kubeone/v1.5/architecture/compatibility/supported-versions/). Support for Kubernetes 1.25 and 1.26 has not been released yet but is in the works and will be released soon.

**Comment by JWebDev** [comment]
@embik Thanks for the quick response. It's a pity.
Already half a year has passed and version 1.26 has already appeared.
